lib/templates/CONTRIBUTING.ejs

```diff
@@ -40,11 +42,11 @@ requested. :sparkles:
 ## Creating a pull request
 
  - Open a pull request, and reference the initial issue in the pull request
-   message (e.g. *fixes #<your-issue-number*). Write a good description and
+   message (e.g. *fixes #<your-issue-number>*). Write a good description and
```

Comment:
Can you please test if `blah -c` renders this correctly? I hope it will not consider it as ejs tag. :balloon: